source/developers/development_environment.markdown

```diff
@@ -22,7 +22,7 @@ $ sudo apt-get install python3-pip python3-dev python3-venv
 
 In order to run `script/setup` below you will need some more dependencies.
 ```bash
-$ sudo apt-get install libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev
+$ sudo apt-get install libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev
```

Comment:
As far as I remember was `libssl-dev` in Jessie shipping `zlib1g-dev`. Seems to be a recent change which will affect Stretch.